boards/espressif/esp32_ethernet_kit/esp32_ethernet_kit-pinctrl.dtsi

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 Grant Ramsay <grant.ramsay@hotmail.com>
+ * Copyright (c) 2022-2025 Grant Ramsay <grant.ramsay@hotmail.com>
```

Comment:
if updating this copyright maybe its better to add Espressif line after the original contributor.